.travis.yml

```diff
@@ -1,9 +1,12 @@
-# openHAB travis.yml
 branches:
   only:
-    - master
-
+  - master
```

Comment:
This statement makes Travis-CI only run against the master. This means [no build will be made](https://docs.travis-ci.com/user/customizing-the-build#sts=Building%20Specific%20Branches) on any other branch of `mozilla/openhab-distro`.

I might not see the big picture, but I wouldn't restricts builds only on master at first.